ffi/java-idsdk/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.iotics.sdk.identity</groupId>
+  <artifactId>java-idsdk</artifactId>
+  <version>1.0-SNAPSHOT</version>
```

Comment:
Is this auto-updated? If not, should we start with `0.1` (I don't know mvn conventions)